clean/test_clean.py

```diff
@@ -0,0 +1,195 @@
+"""Test file for clean.py"""
+
+import pytest
+import os
+from datetime import datetime, timezone, timedelta
+from unittest.mock import MagicMock, patch
+from botocore.exceptions import NoCredentialsError, PartialCredentialsError
+from clean import (s3_connection, lambda_handler)
+
+
+@patch.dict('os.environ', {'AWS_ACCESS_KEY_ID': 'fake_access_key', 'AWS_SECRET_ACCESS_KEY': 'fake_secret_key'})
```

Comment:
you can create a fixture to manage environment variables rather than repeating @patch.dict:

```
@pytest.fixture
def aws_env_vars():
    with patch.dict("os.environ", {"AWS_ACCESS_KEY_ID": "fake_access_key", "AWS_SECRET_ACCESS_KEY": "fake_secret_key"}):
        yield
```